sandboxjs/sandbox_version.json

```diff
@@ -1,4 +1,4 @@
 {
-  "release": "v6.9.0-SNAPSHOT",
-  "version": "6.9.0-SNAPSHOT"
```

Comment:
I think we can leave out this doc because it's for the UI e2e tests downloading sandbox

fyi, @sumengwang @njbriggs

Reply:
We can update it once cdap 6.9.0 is released

Reply:
Correct, we shouldn't change this until the `6.9.0` build is available. And really, if `6.9.1-SNAPSHOT` is ready, we should go right to that.